pkg/apis/messaging/v1beta1/in_memory_channel_defaults.go

```diff
@@ -19,6 +19,13 @@ package v1beta1
 import "context"
 
 func (imc *InMemoryChannel) SetDefaults(ctx context.Context) {
+	// Set the duck subscription to indicate that we support
+	// v1beta1 version of the Subscribable.
+	if imc.Annotations == nil {
+		imc.Annotations = make(map[string]string)
+	}
+	imc.Annotations["messaging.knative.dev/subscribable-version"] = "v1beta1"
```

Comment:
should this be `duck.knative.dev/subscribable` ?

Reply:
sure thing, done.